l10n/po/ja_JP/_guides/kogito-drl.adoc.po

```diff
@@ -93,7 +92,7 @@ msgstr "Gitリポジトリをクローンする： `git clone {quickstarts-clone
 #. type: Plain text
 #: upstream/_guides/kogito-drl.adoc:47
 msgid "The solution is located in the `kogito-drl-quickstart` {quickstarts-tree-url}/kogito-quickstart[directory]."
-msgstr "このソリューションは `kogito-drl-quickstart` {quickstarts-tree-url}/kogito-quickstart[directory] にあります。"
+msgstr "このソリューションは `kogito-drl-quickstart` {quickstarts-tree-url}/kogito-drl-quickstart[directory] にあります。"
```

Comment:
If you noticed a mistake in the upstream, its better to send a pull-request to the upstream to fix the corresponding text in the upstream: 
https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/kogito-drl.adoc#solution